AngularLanguageService/AngularLanguageClient.cs

```diff
@@ -75,13 +77,13 @@ public async Task<Connection> ActivateAsync(CancellationToken token)
             process.StartInfo = info;
             process.ErrorDataReceived += (obj, data) => { outputPane.WriteAsync($"Error from node process: {data.Data}").Forget(); };
 
-            await outputPane.WriteAsync("Starting node process.");
+            outputPane.WriteAsync("Starting node process.").Forget();
 
             try
             {
                 if (process.Start())
                 {
-                    var inputPipe = new Pipe();
+                    var inputPipe = new Pipe(new PipeOptions(useSynchronizationContext: false));
```

Comment:
Is this still needed? I think if you leave this off you're guaranteed to run the I/O operations on the current (background) thread.

Reply:
Nope! Not needed anymore. Removed